src/sentry_clj/tracing.clj

```diff
@@ -0,0 +1,104 @@
+(ns sentry-clj.tracing
+  (:import
+    (io.sentry
```

Comment:
Hi

Can these imports be vectors rather than a list, as in `core`.

Reply:
OK. I'll use vectors, not list.